README.md

```diff
@@ -52,13 +52,24 @@ require one for your use case.
 * [virtualenv](https://pypi.org/project/virtualenv/)
 * [venv](https://docs.python.org/3/library/venv.html)
 
+NOTE: MacOS sometimes has issues with installing since it comes with python2.7 by default.
+
+### via `homebrew`
+
+[brew](https://brew.sh/)
+If you are installing ape on a MacOS. Most users find it easier to install it on brew. It is the same process as pip.
```

Comment:
This first sentence is not a real sentence. It just ends! Maybe you meant to use a comma with the next sentence?